docs-search/client-halogen/src/Docs/Search/BrowserEngine.purs

```diff
@@ -108,14 +107,6 @@ browserSearchEngine =
   , queryModuleIndex: ModuleIndex.queryModuleIndex
   }
 
--- | Find in which part of the index this path can be found.
-getPartId :: List Char -> PartId
-getPartId (a : b : _) =
-  PartId $ (Char.toCharCode a + Char.toCharCode b) `mod` Config.numberOfIndexParts
-getPartId (a : _) =
-  PartId $ Char.toCharCode a `mod` Config.numberOfIndexParts
-getPartId _ = PartId 0
```

Comment:
`getPartId` has been moved to `Docs.Search.Config`, this was the only change necessary so that `Docs.Search.IndexBuilder` did not depend on `Docs.Search.BrowserEngine`.